src/rpc/handlers/LedgerIndex.cpp

```diff
@@ -61,10 +61,9 @@ LedgerIndexHandler::process(LedgerIndexHandler::Input input, Context const& ctx)
         return fillOutputByIndex(maxIndex);
 
     auto const convertISOTimeStrToTicks = [](std::string const& isoTimeStr) {
-        std::tm time = {};
-        std::stringstream ss(isoTimeStr);
-        ss >> std::get_time(&time, DATE_FORMAT);
-        return std::chrono::system_clock::from_time_t(std::mktime(&time)).time_since_epoch().count();
+        std::tm timeStruct;
+        strptime(isoTimeStr.c_str(), DATE_FORMAT, &timeStruct);
```

Comment:
I think `strptime()` is a POSIX function, not from C or C++ standard library. Maybe there is nothing wrong with using it.

I found this one in case you want to try to use something a bit more standard:
https://en.cppreference.com/w/cpp/chrono/c/strftime

Reply:
Thanks. I have checked the strftime, it is to convert the tm to the string.